.github/workflows/validate.yaml

```diff
@@ -0,0 +1,41 @@
+name: Run ocp-build-data-validator
+
+on: 
+  pull_request:
+    branches:
+      - 'openshift-**'
+    paths:
+      - '**.yml'
```

Comment:
If we need to track `rpms/**.yml` and `images/**.yml` we'll need to add those in here as well

Reply:
Right! missed that